### ISSUE TITLE
JSON representation of a work (version)

### ISSUE THREAD

**Issue by srerickson** [psu-libraries/scholarsphere#949]
For API endpoints where work (version) metadata is returned, what should the json structure look like? Something like the following might work. One important principle is to return the version information in the context of the work. Include the most recently published version by default and just provide UUIDS of the other versions. We can include other versions in the version block if requested [?].

```json
{
    "id": "1d7f734e-34dc-4e5f-bb46-6d36d05de481",
    "work_type": "Poster",
    "doi": "10.0...",
    "last_published_version": "uuid2",
    "versions": {
        "uuid2": {
            "id": "uuid",
            "published_state": "published",
            "meta": {},
            "creators": {},
            "files": {},
            "created_at": "",
            "updated_at": ""
        }
    },
    "depositor": {
        "psu_id": "xyz987",
        "dsiplay_name": "Alexis Sullivan",
        "family_name": "Sullivan",
        "given_name": "Alexis",
    },
    "deposit_agreement_version": "2",
    "deposit_agreed_at": "2021...",    
    "deposited_at": "2021...",
    "created_at": "2021-..",
    "updated_at": "2021..."
}
```

**Comment by awead** [comment]
Another idea is to kind of treat this the way we do the resource page, and return a single Work/WorkVersion:

```
{
    "id": "1d7f734e-34dc-4e5f-bb46-6d36d05de481",
    "work_type": "Poster",
    "doi": "10.0...",
    "title": "My Work",
    "description": "....",
    "version": 1
    "latest_version": true,
    "state": "published",
    "creators": {},
    "files": {},
    "depositor": {
        "psu_id": "xyz987",
        "dsiplay_name": "Alexis Sullivan",
        "family_name": "Sullivan",
        "given_name": "Alexis",
    },
    "deposit_agreement_version": "2",
    "deposit_agreed_at": "2021...",    
    "deposited_at": "2021...",
    "created_at": "2021-..",
    "updated_at": "2021..."
}
```

**Comment by srerickson** [comment]
My only hesitation with that json structure is that it combines work-level and version-level information in a way that is hard to follow. For example, is that a work id or a version id? We also have this problem in our`resource/:id` routing.
